arch/risc-v/src/esp32c3/esp32c3_interrupt.S

```diff
@@ -111,7 +111,7 @@ _interrupt_handler:
   csrr a0,  mcause    /* exception cause */
   mv   a1,  sp        /* context = sp */
 
-#if CONFIG_ARCH_INTERRUPTSTACK > 15
+#if CONFIG_ARCH_INTERRUPTSTACK > 3
```

Comment:
Nice finding, the current code from master is wrong.
And I've found several other places where the interrupt stack is being handled with wrong alignment.
I'll prepare a new PR for this.

Reply:
What's the issue here?  This just checks that we don't have an empty interrupt stack.

Reply:
`CONFIG_ARCH_INTERRUPTSTACK` needs to be checked against its minimum required size, which for RISC-V is 16 bytes.

Reply:
Yes, but for what follows both 15 or 3 are not enough.  I personally used to read this as "non empty" discarding any alignment.

Reply:
@jerpelea The `> 15` is already correct, you may revert this one also.